src/rules.jl

```diff
@@ -518,6 +530,42 @@ function lrp!(Rᵏ, rule::GeneralizedGammaRule, layer, modified_layers, aᵏ, R
     @. Rᵏ = aᵏ⁺ * (cˡ⁺ + cʳ⁻) + aᵏ⁻ * (cˡ⁻ + cʳ⁺)
 end
 
+"""
+    LayerNormRule(affine_rule=ZeroRule())
+
+LayerNormRule rule.
+
+# Optional arguments
+- `affine_rule`: Rule applied to affine transformation, defaults to `ZeroRule()`.
+
+# References
+- 
+"""
+struct LayerNormRule{T <: AbstractLRPRule} <: AbstractLRPRule
+    affine_rule::T
+end
+
+LayerNormRule() = LayerNormRule(ZeroRule())
+
+function lrp!(Rᵏ, rule::LayerNormRule, layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
+    # forward pass: split in normalization and scale
+    # aᵏ ->(normalize) aᵏₙ ->(scale) aᵏ⁺¹
+    ## normalize
+    eps = convert(float(eltype(aᵏ)), layer.ϵ)
+    μ = mean(aᵏ; dims = 1:length(layer.size))
+    σ = std(aᵏ, dims=1:length(layer.size), mean=μ, corrected=false)
+    z = aᵏ - μ
+    aᵏₙ = @. z / (σ + eps)
+    # lrp pass
+    ## Rᵏ⁺¹ ->(scale) Rᵏₙ ->(normalize) Rᵏ
+    ## scale: call LRP on affine layer with "subrule" rule.affine_rule
+    Rᵏₙ = similar(aᵏₙ)
+    lrp!(Rᵏₙ, rule.affine_rule, layer.diag, modify_layer(rule.affine_rule, layer.diag), aᵏₙ, Rᵏ⁺¹)
```

Comment:
Just write into `Rᵏ` twice to save some allocations.
(This little trick should be documented.)

Reply:
done :)